<launch>
  <node
    pkg="autocycle"
    type="start_nav"
    name="navigation_core"
    output="screen"
    required="true"
  />
  <node
    pkg="autocycle"
    type="run_lidar"
    name="run_lidar"
    output="screen"
    required="true"
  />
  <node
    pkg="autocycle"
    type="parse_lvx"
    name="parse_lvx_file"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="bezier_visualization"
    name="bezier"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="get_roll"
    name="get_roll"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="fix_roll"
    name="fix_roll"
    output="screen"
    respawn="true"
  />
</launch>
